src/analysis/Environments/Environments.test.ts

```diff
@@ -892,7 +892,7 @@ describe('Environments Component', () => {
   //   );
   // });
   describe('onEvent', () => {
-    it('calls onEvent[dataRefesh] with a runtime', async () => {
+    it.skip('calls onEvent[dataRefesh] with a runtime', async () => {
```

Comment:
Why is this test skipped?

Reply:
Ah, this should have been labelled as a draft, I was just poking around with some different things and didn't want to run the tests myself locally. Closing now.